File: parte001/ex01_area_rectangulo.java
import java.util.Scanner;

public class ex01_area_rectangulo {
    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);

        System.out.print("Introduzca la base del rectángulo: ");
        double base = teclado.nextDouble();
        teclado.nextLine();
        
        System.out.print("Introduzca la altura del rectángulo: ");
        double altura = teclado.nextDouble();
        teclado.nextLine();

        double area = base * altura;
        System.out.println("El área del rectángulo es: " + area);

        double perimetro = 2 * (base + altura);
        System.out.println("El perímetro del rectángulo es: " + perimetro);

        teclado.close();
    }
}

File: parte001/ex01_area_rectangulo.cpp
#include <iostream>

using namespace std;

int main() {
    double base, altura, area, perimetro;

    cout << "Introduzca la base del rectangulo: ";
    cin >> base;
    cout << "Introduzca la altura del rectangulo: ";
    cin >> altura;

    area = base * altura;

    cout << "El area del rectangulo es: " << area << endl;

    perimetro = 2 * (base + altura);

    cout << "El perimetro del rectangulo es: " << perimetro << endl;

    return 0;
}